tests/ui/suspicious_unary_op_formatting.rs

```diff
@@ -0,0 +1,23 @@
+#![warn(clippy::suspicious_unary_op_formatting)]
+
+#[rustfmt::skip]
+fn main() {
+    // weird binary operator formatting:
+    let a = 42;
+
+    if a >- 30 {
+    } else if a >=- 30 {
+    }
+
+    let b = true;
+    let c = false;
+
+    if b &&! c {
+    }
+
+    // those are ok:
+    if a >-30 {
+    } else if a < -30 {
+    } else if b && !c {
+    }
```

Comment:
Can you add another test with more than one space between the unop and the operand?
```suggestion
    }
    
    if a >-   30 {}
    if a > -   30 {}
```

Reply:
Sure! Added your suggestions.